Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/LoginController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Benchmark.WebApi.Interfaces;
+using Benchmark.WebApi.Models.Database;
+using Benchmark.WebApi.Models.Response.Overview;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Benchmark.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [Authorize]
+    [ApiController]
+    public class LoginController : BaseUtilityController
+    {
+        private readonly IEmployeeService _employeeService;
+
+        public LoginController(IEmployeeService employeeService)
+        {
+            _employeeService = employeeService;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IList<LoginOverview>>> GetAllLogins()
+        {
+            return ToActionResult(await _employeeService.GetAllLogins());
```

Comment:
Probs fine for now to have it in the employee service but if this controller gets any bigger we should probably move it to its own service.

edit: I see that it has its own repo, could a loginService be made?

```suggestion
            return ToActionResult(await _loginService.GetAllLogins());
```